project/src/main/webapp/script.js

```diff
@@ -152,17 +151,23 @@ function createPlaceElement(place) {
   const name = document.createElement('li');
   name.innerText = place.name;
   placeElement.appendChild(name);
+  placeElement.appendChild(document.createElement('br'));
   // Add link to website
-  if (place.websiteUrl) {
-    addLinkToPlaceElement(placeElement, place.websiteUrl, 'Website');
-  }
-  if (place.googleUrl) {
-    addLinkToPlaceElement(placeElement, place.googleUrl, 'Google Maps link');
-  }
-  if (!place.websiteUrl && !place.googleUrl) {
-    placeElement.appendChild(document.createElement('br'));
+  let hasWebUrl = place.websiteUrl;
```

Comment:
I don't really like this naming since `hasWebUrl` is actually a string and not a boolean value. 
Maybe make it more explicit by something like
`let hasWebUrl = place.websiteUrl != null`?

Reply:
sure, good idea. done